ChangeLog

```diff
@@ -304,6 +304,10 @@ Release date: TBA
 
   Closes #4369 and #6523
 
+* Fix bug where it writes a plain text error message to stdout, invalidates output formats.
+
+  Closes #6597
+
```

Comment:
And could you copy the changelog to 2.14.rst?
```suggestion
* Fix bug where it writes a plain text error message to stdout, invalidating output formats.
```

Reply:
Done

Reply:
(missed the commit suggestion, now fixed manually)